lib/menu.rb

```diff
@@ -0,0 +1,43 @@
+class Menu
+
+  attr_reader :dishes
+
+  BLANKS = 29
+
+  def initialize
+    @dishes = import_dishes
+  end
+
+  def dishes
+     @dishes.clone
+  end
+
+  def show
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for show is too high. [17.03/15] (http://c2.com/cgi/wiki?AbcMetric)